packages/design-system/src/components/Form/Field/Input/Input.File.tsx

```diff
@@ -234,7 +230,11 @@ const InputFile = React.forwardRef((props: FileProps, ref: React.Ref<HTMLInputEl
 							</span>
 						</div>
 					) : (
-						<div className="input-file__preview preview">
+						<div
+							className={classnames('input-file__preview preview', {
+								singleFile: Array.from(files).length === 1,
```

Comment:
why Array.from ?

Reply:
I used the syntax provided for the file listing. IIRC, `files` isn't an array in some cases?

Reply:
https://attacomsian.com/blog/javascript-iterate-filelist yup, not an actual Array.